src/components/dropdownSearch/MultiSelectDropdownSearch/MultiSelectDropdownSearch.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { DropdownSearch } from '../DropdownSearch';
+import {MultiselectInput} from './MultiSelectInput';
+
+export class MultiSelectDropdownSearch extends DropdownSearch {
+
+  protected getNoOptions(): JSX.Element[] {
+    if (this.props.filterText.length > 0 && !this.props.selectedOptions.containsElementWithProperties({displayValue: this.props.filterText})) {
+      return [
+        <li key='noResultDropdownSearch' onMouseDown={() => this.props.onCustomOptionClick(this.props.filterText)}>
+          <span>{`Create option for "${this.props.filterText}"`}</span>
+        </li>,
+      ];
+    }
+    return super.getNoOptions();
+  }
+
+  render() {
+    return (
+      <div className={this.getClasses()} style={this.getStyles()}>
+        <MultiselectInput selectedOptions={this.props.selectedOptions.getQueue()}
```

Comment:
Add a line break before selectedOptions and move your indentation way to the left.